### PR TITLE
WIP: Fixing issue #132. actions/name?blocking=true&result=true returns the result

### PR DESCRIPTION
**Pull request by fxulusoy**

### CODE REVIEW

**Comment by markusthoemmes** [comment]
As this is an additional parameter for the API, update swagger documentation?

**Comment by rabbah** [comment]
All API changes must have accompanying functional tests. You can add to one of the blocking invoke tests in Actions API Tests. Should make sure to test for both successful, and failed invoked.

File: core/controller/src/whisk/core/controller/Actions.scala [hunk 1]
```diff
@@ -246,12 +246,13 @@ trait WhiskActionsApi extends WhiskCollectionAPI {
                             case Success((activationId, None)) =>
                                 complete(Accepted, activationId.toJsObject)
                             case Success((activationId, Some(activation))) =>
+                                val response = if (result) activation.getResultJson else activation.toExtendedJson
```
Comment (rabbah) [inline core/controller/src/whisk/core/controller/Actions.scala]: In addition to updating the swagger, please update the javadoc for this method.

**Comment by rabbah** [comment]
Added unit tests and updated swagger in #432.

**Pull request closed without merging**